packages/editor-ui/src/components/Telemetry.vue

```diff
@@ -34,6 +34,10 @@ const isTelemetryEnabled = computed((): boolean => {
 	return !!telemetry.value?.enabled;
 });
 
+const selfInstallSrc = computed((): string => {
+	return `https://n8n.io/self-install?instanceId=${rootStore.instanceId}&userId=${currentUserId.value}`;
```

Comment:
why do we need the user-id here?

Reply:
Requested by David here: https://linear.app/n8n/issue/PAY-1671/enable-linking-instance-to-website-activity#comment-755e1f09

Reply:
That comment says 
> send the instance user ID rather than the instance ID

I don't know if we still need both the ids here. but I don't want to block this either.

Reply:
Agree, strictly speaking they misspoke, but they do need both. User ID without instance ID would not be useful.